Unit-01/03-templating/app.py

```diff
@@ -0,0 +1,32 @@
+from flask import Flask, render_template, request
+
+app = Flask(__name__)
+
+# PART 1
+@app.route('/person/<name>/<int:age>')
+def age(name, age):
+	return render_template('name.html', name = name, age = age)
```

Comment:
just nitpicking, but it's considered more "pythonic" to not put a space between kwarg operators, i.e. 

```python
render_template('name.html', name=name, age=age)
```